.editorconfig

```diff
@@ -0,0 +1,9 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
```

Comment:
Personally, I wouldn't specify `crlf` or `lf` for `end_of_line` and just leave that to the editor to autodetect from what git has checked out. However, we don't have a `.gitattributes` set up properly on this repository yet.